packages/benchmarks/lit-html/stub1/tachometer.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Polymer/tachometer/master/config.schema.json",
+  "sampleSize": 30,
+  "timeout": 0,
+  "benchmarks": [
+    {
+      "url": "lit-html/stub1/stub1.html",
+      "browser": {
+        "name": "chrome",
+        "headless": true
+      },
+      "measurement": {
+        "mode": "performance",
+        "entryName": "render"
+      },
+      "expand": [
+        {
+          "name": "this-change"
+        },
+        {
+          "name": "tip-of-tree",
+          "packageVersions": {
+            "label": "tip-of-tree",
+            "dependencies": {
+              "lit-html": {
+                "kind": "git",
+                "repo": "https://github.com/Polymer/lit-html.git",
+                "ref": "lit-next",
+                "subdir": "packages/lit-html",
+                "setupCommands": [
+                  "npm ci",
+                  "npm run bootstrap",
+                  "npm run build"
+                ]
+              },
+              "lit-element": {
```

Comment:
don't need this, right?

Reply:
I think we need those at least temporarily right now because the package.json for `packages/benchmarks` currently includes an unreleased lit-element version 3.0.0-pre (which when installed during lerna bootstrap is fine), but when re-installed via `packageVersions` by tachometer would error if it weren't replaced with a good version (even if it is unused by the benchmark).  I hit this in working on top of this PR last night.

Reply:
Exactly! Kind of annoying. I was thinking tachometer could check for symlinks in `node_modules`, and convert those into a local file path version for NPM whenever doing a dependency swap.